Annex60/Fluid/BaseClasses/PartialThreeWayResistance.mo

```diff
@@ -78,87 +76,44 @@ partial model PartialThreeWayResistance
     final X_start=X_start,
     final C_start=C_start,
     final allowFlowReversal=true,
-    final prescribedHeatFlowRate=false) if
-       dynamicBalance "Fluid volume to break algebraic loop"
+    final prescribedHeatFlowRate=false) "Fluid volume to break algebraic loop"
     annotation (Placement(transformation(extent={{-10,0},{10,20}})));
 
-protected
```

Comment:
It's more a question of programming style: I prefer equations to be conditionally removed when a component is conditional, rather than making the equations themselves conditional. Also this port forces the mixing location to this point, but this should not lead to different equations in this case.
Since the parameter `dynamicBalance` is removed, `port_internal` serves no more use so it can be removed in my opinion. Thanks for verifying!